tests/test_app.py

```diff
@@ -86,26 +188,50 @@ def test_search_results_calls_search_api(self, mock_post):
         self.assertTrue(str(query_dict['street']) in str(kwargs['data']))
         self.assertTrue(str(query_dict['paon']) in str(kwargs['data']))
 
-    @mock.patch('requests.post', return_value=single_response)
-    def test_single_result(self, mock_post):
+    @mock.patch('service.server.get_property_address', return_value=one_DB_result)
+    @mock.patch('requests.post', return_value=single_PPI_response)
+    def test_get_property_returns_data_from_PPI_API_when_single_result(self, mock_post, mock_get_property_address):
+        search_query = "PL2%201AD/ALBERT%20ROAD_10_FLAT%202"
+        response = self.app.get('/properties/%s' % search_query)
+
+        self.assertTrue('"amount": "100000"' in str(response.data))
+        self.assertTrue('"date": "2003-04-17"' in str(response.data))
+
+    @mock.patch('service.server.get_property_address', return_value=one_DB_result)
+    @mock.patch('requests.post', return_value=double_PPI_response)
+    def test_get_property_returns_first_PPI_API_result_when_more_than_one(self, mock_post, mock_get_property_address):
         search_query = "PL2%201AD/ALBERT%20ROAD_10_FLAT%202"
         response = self.app.get('/properties/%s' % search_query)
 
-        self.assertTrue(str('100000') in str(response.data))
-        self.assertTrue(str('2003-04-17') in str(response.data))
+        self.assertTrue('"amount": "100001"' in str(response.data))
+        self.assertTrue('"date": "2003-04-18"' in str(response.data))
 
-    @mock.patch('requests.post', return_value=double_response)
-    def test_two_result(self, mock_post):
+    @mock.patch('service.server.get_property_address', return_value=multiple_DB_results)
+    def test_get_property_returns_500_error_when_the_DB_returns_two_result(self, mock_get_property_address):
         search_query = "PL6%208RU/PATTINSON%20DRIVE_100"
         response = self.app.get('/properties/%s' % search_query)
 
-        self.assertTrue(str('More then one record found.') in str(response.data))
+        self.assertTrue('More than one record found' in str(response.data))
+
+    @mock.patch('service.server.get_property_address', return_value=one_DB_result)
+    @mock.patch('requests.post', return_value=empty_PPI_response)
+    def test_get_property_returns_no_PPI_data_when_PPI_API_returns_empty_result(self, mock_post, mock_get_property_address):
+        search_query = "PL2%201AD/ALBERT%20ROAD_10_FLAT%202"
+        response = self.app.get('/properties/%s' % search_query)
 
-    @mock.patch('requests.post', return_value=double_response)
-    def test_unable_to_split(self, mock_post):
+        self.assertTrue('"amount": ""' in str(response.data))
+        self.assertTrue('"date": ""' in str(response.data))
```

Comment:
Would it be possible to use assertIn in those cases? It makes the error message nice instead 'should be True but it's False' not-super-helpful error message you get with assertTrue.